index.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 // TODO: Use the `URL` global when targeting Node.js 10
-const URLParser = typeof URL === 'undefined' ? require('url').URL : URL;
+const URL = require('url').URL; // eslint-disable-line prefer-destructuring
 
 function testParameter(name, filters) {
 	return filters.some(filter => filter instanceof RegExp ? filter.test(name) : filter === name);
 }
 
 module.exports = (urlString, opts) => {
 	opts = Object.assign({
+		defaultProtocol: 'http:',
 		normalizeProtocol: true,
-		normalizeHttps: false,
-		normalizeHttp: false,
-		stripFragment: true,
+		forceHttp: false,
+		forceHttps: false,
+		stripHash: true,
```

Comment:
I'm ok with this renaming, but make it backward compatible by adding some aliases to the old names below. I don't want to do a major release just for this. I just recently did a major release.

Reply:
:crying_cat_face: Ok... :P

Reply:
Done.